coalib/bears/Bear.py

```diff
@@ -47,6 +49,67 @@ class Bear(Printer, LogPrinter):
     >>> class SomeBear(Bear):
     ...     LANGUAGES = "Java"
 
+    To indicate the requirements of the bear, give to the ``REQUIREMENTS``
+    tuple instances of the ``PackageRequirement`` class:
+
+    >>> class SomeBear(Bear):
+    ...     REQUIREMENTS = (
+    ...             PackageRequirement('pip', 'coala_decorators', '0.2.1'),)
+
+    If your bear uses requirements from a manager we have a subclass from,
+    you can use the subclass, such as ``PythonRequirement``, without specifying
+    manager:
+
+    >>> class SomeBear(Bear):
+    ...     REQUIREMENTS = (PythonRequirement('coala_decorators', '0.2.1'),)
+
+    Alternatively, you should use the ``multiple`` method if you have more
+    requirements from the same manager. This can receive both tuples of strings,
+    in case you want a specific version, or a simple string, in case you want
+    the latest version to be installed.
+
+    This is the case where you would provide strings only, to install the
+    latest version automatically:
+
+    >>> class SomeBear(Bear):
+    ...     REQUIREMENTS = PythonRequirement.multiple('coala_decorators',
+    ...                                               'setuptools')
+
+    And if you choose to mix them, specifying version for some and for some not:
+
+    >>> class SomeBear(Bear):
+    ...     REQUIREMENTS = PythonRequirement.multiple('coala_decorators',
+    ...                                            ('setuptools', '19.2'))
+
+    In case you provide too many arguments into the tuple, an error will be
+    raised:
+
+    >>> class SomeBear(Bear):
+    ...     REQUIREMENTS = PythonRequirement.multiple('coala_decorators',
+    ...                                 ('setuptools', '19.2', 'colorama'))
```

Comment:
I want to see this whole thing in action, perhaps with a more thorough integration test. Then the inconsistencies might clean themselves up.